packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -245,13 +245,36 @@ export const visualizeSharedTree: VisualizeSharedObject = async (
 	visualizeChildData: VisualizeChildData,
 ): Promise<FluidObjectTreeNode> => {
 	const sharedTree = sharedObject as ISharedTree;
-	const content = sharedTree.contentSnapshot();
+	const contentSnapshot = sharedTree.contentSnapshot();
+
+	/**
+	 * Root node of the SharedTree's treeview. Assume there is only one root node.
+	 */
+	const treeView = contentSnapshot.tree[0];
+
+	/**
+	 * Schema of the tree node.
+	 */
+	const treeSchema = contentSnapshot.schema.nodeSchema.get(treeView.type);
+
+	/**
+	 * Traverses the SharedTree and generates a visual representation of the tree (mainly composed of `schema` and `fields` field).
+	 */
+	const visualTreeRepresentation = visualizeSharedTreeNodeBySchema(
```

Comment:
Verify whether to pass in `treeView.type` when first calling `visualizeSharedTreeNodeBySchema`.